docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN npm install npm@latest --location=global \
     && PUBLIC_URL=/static/reactapp/ npm run build
 
 # Stage 2: Backend
-FROM python:3.9.16 as backend-build
+FROM python:3.10.0 as backend-build
```

Comment:
you cannot just upgrade the version of this without checking possible code issues. Plus, it makes no sense to update to the 0 patch version, use the latest.
Why do you need to have python 3.10?

Reply:
Somehow, after adding the installation scripts to the dockerfile as well,  zippy wasn't able to import properly.
I performed an ssh into the container just to find out that every time i call zippy in the cli, this error is thrown:
```
root@bf217b07b610:~# zippy
Traceback (most recent call last):
  File "/usr/local/bin/zippy", line 5, in <module>
    from zippy.zippy import main
  File "/usr/local/lib/python3.9/site-packages/zippy/__init__.py", line 1, in <module>
    from .zippy import *
  File "/usr/local/lib/python3.9/site-packages/zippy/zippy.py", line 15, in <module>
    from typing import List, Optional, Tuple, TypeAlias
ImportError: cannot import name 'TypeAlias' from 'typing' (/usr/local/lib/python3.9/typing.py)
```
Upon further investigations, I found out that TypeAlias is available for Python3.10.0 and above
https://docs.python.org/3/whatsnew/3.10.html#pep-613-typealias

Reply:
ok but the problem is that:
* first, the last patch release is required, as already stated.
* second, updating the base image python version *must* be done in a separate PR, to track and test it properly, not inside a random feature PR like this one. This is very important. You could break a lot of other things if this is not handled correctly.